autoload/slime/common.vim

```diff
@@ -35,7 +41,12 @@ function! slime#common#capitalize(text)
 endfunction
 
 function! slime#common#system(cmd_template, args, ...)
-  let escaped_args = map(copy(a:args), "shellescape(v:val)")
+  if &l:shell !=# "cmd.exe"
+      let escaped_args = map(copy(a:args), "shellescape(v:val)")
+  else
+      let escaped_args = a:args
+  endif
```

Comment:
what's the story here?

is there never a need to escape for `cmd.exe`?

Reply:
Rather, the escapes break cmd. It passes the literal quotes to the application and doesn't interpret them.

Sometimes. Maybe. :facepalm:

Anyway, this got it to work.